test/functional/inspec_vendor_test.rb

```diff
@@ -53,10 +61,51 @@
     out.stderr.must_equal ''
   end
 
+  it 'ensure json command is not fetching remote profiles if vendored' do
```

Comment:
A good test for this would be to check that the vendored profile(ex: 01414bd307ea2f7d4dc8cd141085ba7ad61d4c3b2606d57b2dae987c1c3954cb.tar.gz was copied in the cache) instead of a remote one